src/yb/client/client.cc

```diff
@@ -2295,6 +2295,13 @@ Result<bool> YBClient::CheckIfPitrActive() {
   return data_->CheckIfPitrActive(deadline);
 }
 
+Result<ListTablesResponsePB> YBClient::ListTableInfo() {
```

Comment:
Probably ok for the feature branch. But, you are duplicating the logic between this function and the one below.

You could merge the commonality between the two to reduce future maintenance. Add a todo if you aren't fixing it now, but will get to it when merging to the master branch.